src/main/python/camdkit/framework.py

```diff
@@ -334,24 +448,34 @@ def from_json(self, json_dict: dict):
           raise ValueError
 
   @classmethod
-  def make_json_schema(cls) -> dict:
-    schema = {
-      "$schema": "https://json-schema.org/draft/2020-12/schema",
-      "type": "object",
-      "properties": {}
-    }
+  def make_json_schema(cls, is_root=False) -> dict:
+    schema = {}
+    if is_root:
```

Comment:
Why is this needed?

The framework current assumes a flat model where a `ParameterContainer` contains multiple `Parameter` but cannot contain other `ParameterContainer`.

Reply:
Ah yes this is a leftover from when I implemented recursive nesting here - will remove.